server/backend/TextToSpeech/YngpingAudioSynthesizer.cs

```diff
@@ -35,10 +35,10 @@ public sealed class YngpingAudioSynthesizer
             var audioFiles =
                 audioCodes.Select(code => Path.Combine(this.ttsAudioFolder, code + ".wav"))
                     .ToArray();
-            return TrimSilence(audioFiles);
+            return TrimAudio(audioFiles);
```

Comment:
A more descriptive name for `TrimAudio` might be `ConcatSyllables` or `ConcatSyllableAudio`?